src/app/pages/system/certificates/certificate-list/certificate-list.component.ts

```diff
@@ -80,16 +81,21 @@ export class CertificateListComponent {
         label: helptext_system_certificates.list.action_export_certificate,
         onClick: (row) => {
           this.ws.call('certificate.query', [[["id", "=", row.id]]]).subscribe((res) => {
+            const fileName = res[0].name + '.crt';
             if (res[0]) {
-              this.ws.call('core.download', ['filesystem.get', [res[0].certificate_path], res[0].name + '.crt']).subscribe(
+              this.ws.call('core.download', ['filesystem.get', [res[0].certificate_path], fileName]).subscribe(
                 (res) => {
-                  this.snackBar.open(helptext_system_certificates.list.snackbar_open_window_message, helptext_system_certificates.list.snackbar_open_window_action, {
-                    duration: 5000
+                  const url = res[1];
+                  const mimetype = 'application/gzip';
```

Comment:
These files aren't of mimetype application/gzip, I "think" the right one is application/x-x509-user-cert

https://www.sitepoint.com/mime-types-complete-list/